docs/source/tutorial.rst

```diff
@@ -117,56 +123,112 @@ build, run the following inside vagrant:
 
   $ /project/sawtooth-core/bin/build_all
 
-Running validator
-=================
+Validator Start-up Process
+==========================
+
+Create Genesis Block
+--------------------
+
+In most use cases, it is not necessary to create a genesis block when starting
+a validator, because the validator joins an existing distributed ledger
+network. However, as a developer, you may often need to create short-lived
+test networks. In this case, you need to create a genesis block when
+instantiating a new network.
+
+The genesis block contains some initial values that are necessary when a
+Sawtooth Lake distributed ledger is created and used for the first time.
 
-To start a validator, log in to the development environment with 'vagrant ssh'
+To create the genesis block, log in to the development environment with the
+command `vagrant ssh` and run the following command:
+
+.. code-block:: console
+
+  $ sawtooth admin genesis
+  Generating /home/ubuntu/sawtooth/data/genesis.batch
+
+
+.. note:: 
+
+  If you need to delete previously existing block-chain data before running a
+  validator in the vagrant environment, simply run the following command: 
+  `rm /home/ubuntu/sawtooth/data/*`
+
+
+Start Validator
+---------------
+
+To start a validator, log in to the development environment with `vagrant ssh`
 and run the following commands:
 
 .. code-block:: console
 
    $ cd sawtooth-core/
-   $ validator -v
+   $ validator -vv
```

Comment:
I think we are missing a step, with the current master when I run this command I get:

ubuntu@ubuntu-xenial:/project/sawtooth-core$ validator -vv
[17:00:56.127 INFO     path] Skipping path loading from non-existent config file: /home/ubuntu/sawtooth/etc/path.toml
[17:00:56.127 INFO     cli] config [path]: config_dir = "/home/ubuntu/sawtooth/etc"
[17:00:56.127 INFO     cli] config [path]: key_dir = "/home/ubuntu/sawtooth/keys"
[17:00:56.127 INFO     cli] config [path]: data_dir = "/home/ubuntu/sawtooth/data"
[17:00:56.127 INFO     cli] config [path]: log_dir = "/home/ubuntu/sawtooth/logs"
[17:00:56.127 ERROR    cli] No such signing key file: /home/ubuntu/sawtooth/keys/validator.wif
[17:00:56.127 ERROR    cli] Initialization errors occurred (see previous log ERROR messages), shutting down.

Running this command is resolved the issue, Not sure what the intention is for validator key generation

sawtooth keygen --key-dir /home/ubuntu/sawtooth/keys/ validator